examples/CommandBasedRobot/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -69,27 +70,32 @@ public void manualDrive(XboxController m_gamepad) {
     // Pass the correct joystick properties of whichever chooser option is selected.
     switch (oiChooser.getSelected()) {
       case TWO_JOYSTICK_GAS:
-        xDirection = m_gamepad.getX(Hand.kLeft);
-        yDirection = -m_gamepad.getY(Hand.kLeft);
+        xDirection = applyDeadband(m_gamepad.getX(Hand.kLeft));
+        yDirection = applyDeadband(-m_gamepad.getY(Hand.kLeft));
+        vector = Math.sqrt(xDirection * xDirection + yDirection * yDirection);
         rotationSpeed = applyDeadband(m_gamepad.getX(Hand.kRight));
         translationSpeed = m_gamepad.getTriggerAxis(Hand.kRight);
+        break;
       case SPLIT_TRIGGER:
-        xDirection = m_gamepad.getX(Hand.kRight);
-        yDirection = -m_gamepad.getY(Hand.kLeft);
+        xDirection = applyDeadband(m_gamepad.getX(Hand.kRight));
+        yDirection = applyDeadband(-m_gamepad.getY(Hand.kLeft));
+        translationSpeed = Math.sqrt(xDirection * xDirection + yDirection * yDirection);
```

Comment:
I think that we should revert back to using the max calculation for split trigger. Unlike with the single joystick controls, the 2 axes aren't mechanically linked, so this calc actually maxes out at `sqrt(2)`.
```suggestion
        translationSpeed = Math.max(Math.abs(xDirection), Math.abs(yDirection));
```